manim/animation/updaters/mobject_update_utils.py

```diff
@@ -56,8 +56,32 @@ def updater(mob):
 
 
 def always_redraw(func):
+    """Redraw a mobject every frame.
+
+    .. manim:: TangentAnimation
+
+        class TangentAnimation(Scene):
+            def construct(self):
+                ax = Axes()
+                sine = ax.plot(np.sin, color=RED)
+                alpha = ValueTracker(0)
+                point = always_redraw(
+                        lambda: Dot(
+                            sine.point_from_proportion(alpha.get_value()),
+                            color=BLUE)
+                    )
+                tangent = always_redraw(
+                    lambda: TangentLine(
+                        sine,
+                        alpha=alpha.get_value(),
+                        color=YELLOW,
+                        length=4)
+                )
+                self.play(Create(ax), Create(sine), Create(point), Create(tangent))
+                self.play(alpha.animate.set_value(1), rate_func=linear, run_time=3)
```

Comment:
Looking at the latest RTD build logs for this PR (https://readthedocs.org/api/v2/build/21513650.txt), this example currently adds about 7 seconds of render time to the documentation, which is (compared to other examples) a bit much.

I suggest to reduce the rendered length of the example a bit further; for example by removing the creation part of the animation and further reducing the run time of the actual animation. It would be good to get the render time below one second (although this is more of a personal preference rather than a set-in-stone rule that we have).

```suggestion
                self.add(ax, sine, point, tangent)
                self.play(alpha.animate.set_value(1), rate_func=linear, run_time=2)
```